site/src/js/index.js

```diff
@@ -1,90 +1,101 @@
 /**
  * Initialising global variables
  */
-let contentEl = document.querySelector('#content'),
-    filterInput = document.querySelector('#filter'),
-    sortingInput = document.querySelector('#sorting'),
-    firstLoad = true,
-    selector = new Selectr('#tags', {
-        multiple: true,
-        placeholder: 'Choose tags...',
-        data: window.swagTags.map(tag => ({value: tag, text: tag}))
+const ACTIVE_CLASS = 'visible';
+const sort = {
+    ASC_A: (a,b) => a.dataset.name > b.dataset.name ? 1 : -1,
+    DSC_A: (a,b) => a.dataset.name < b.dataset.name ? 1 : -1,
+    ASC_D: (a,b) => a.dataset.difficulty > b.dataset.difficulty ? 1 : -1,
+    DSC_D: (a,b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
```

Comment:
Let's be explicit in naming, like `ALPHABETICAL_ASC`, `DIFFICULTY_DESC`, etc.

Reply:
Done 😄